src/parameterized_models.jl

```diff
@@ -48,6 +48,44 @@ end
 survival(m::Makeham,age) = exp(-cumhazard(m,age))
 
 
+"""
+    AlterMakeham(;μ,σ,c)
```

Comment:
I think `Makeham2` would be more consistent with the rest of the names. Same with `AlterGompertz`